migrations/transformWordpressToUrlSlug.js

```diff
@@ -0,0 +1,29 @@
+// Copy the wordpress URL field into the urlSlug field for all blog posts
+// that have a wordpress URL set, and no Url Slug set.
+// Should be run with the contentful migration tool: https://github.com/contentful/contentful-migration
```

Comment:
Oh, this is interesting and looks relatively easy to write..I usually just told them "no" to migrate data in Contentful—gave David/etc a list of ids instead—but this is _everything_, so a bit different…

👍 Just don't mess it up! 😃 

Ideally when we upgrade, we can use environments properly in Contentful and this will be less scary than it is here.